test/struct.js

```diff
@@ -379,4 +379,29 @@ describe('Struct', function () {
 
   })
 
+  describe('usePack()', function () {
+    var size = {
+        unpack: { ia32: 8, x64: 16 },
```

Comment:
Ya I think you're right, we should go the native route. Add the struct definition, packed, to the C++ test file, and then export the sizeof and alignof to compare when packed.